converter/java/build.gradle

```diff
@@ -42,4 +40,16 @@ java {
     toolchain {
         languageVersion = JavaLanguageVersion.of(17)
     }
-}
\ No newline at end of file
+}
+
+task compileWrapper(type: Exec) {
+    doFirst {
+        if (System.properties['os.name'].toLowerCase().contains('windows')) {
+            executable "./resources/compile.bat"
+        } else {
+            executable "./resources/compile.sh"
+        }
+    }
+}
+
+compileJava.dependsOn compileWrapper
```

Comment:
please configure your IDE to auto-add newlines - these get in a way when reviewing. Thx :)

Reply:
👍🏼 done